client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -246,13 +255,15 @@ class EmailProvidersComparison extends React.Component {
 	renderGSuiteDetails( className ) {
 		const { currencyCode, gSuiteProduct, translate } = this.props;
 
-		const logo = config.isEnabled( 'google-workspace-migration' )
```

Comment:
The logic cleanup here would be good to carry through to the new comparison page from #51307 as well, but I think that can be in a separate PR.

Reply:
Gotcha, I'll add a card to take care of that.